src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -35,11 +37,12 @@ public void loadSomeData() {
 					List<Map> placeCategories = (List<Map>) venue.get("categories");
 					
 					count++;
-					scanner.close();
+					
+					System.out.println("bla bla bla");
 				} catch (final FileNotFoundException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS00108)